emcc.py

```diff
@@ -2435,11 +2435,11 @@ def emterpretify(js_target, optimizer, options):
 
 def emit_js_source_maps(target, js_transform_tempfiles):
   logging.debug('generating source maps')
-  jsrun.run_js(shared.path_from_root('tools', 'source-maps', 'sourcemapper.js'),
-               shared.NODE_JS, js_transform_tempfiles +
-               ['--sourceRoot', os.getcwd(),
-                '--mapFileBaseName', target,
-                '--offset', str(0)])
+  run_process_js(shared.path_from_root('tools', 'source-maps', 'sourcemapper.js'),
```

Comment:
`run_process_js` is kind of an odd name. Would it work to move the function to the `jsrun` module, then it can be `jsrun.process_js`?

Regardless of naming, it's a bit unintuitive that we'd have have a module that runs JS, and then a fairly separate way to run JS in shared. Should this function not live in jsrun?

Reply:
The thing is the runjs.py module is mostly designed to run generated JS during testing.   I hear your feedback though and I move this function into that file there for now.  I willfollowup with a another refactor to move most of jsrun into the tests runner.